packages/signal-polyfill/src/wrapper.spec.ts

```diff
@@ -484,11 +484,11 @@ describe("Errors", () => {
 
 describe("Cycles", () => {
   it("detects trivial cycles", () => {
-    const c = new Signal.Computed(() => c.get());
+    const c: Signal.Computed<never> = new Signal.Computed(() => c.get());
```

Comment:
did something go wrong with inference here?

Reply:
Yes it's an infinite loop c is depending on itself. TS wasn't being ran in strict mode before this PR, as the tests are excluded from the main tsconfig.